addons/uh60_jvmf/XEH_preStart.sqf

```diff
@@ -1,3 +1,11 @@
 #include "script_component.hpp"
 
 #include "XEH_PREP.hpp"
+
+[
+	"vtx_uh60_jvmf_defaults"
+	,"CHECKBOX"
+	,["Enable JVMF Default Messages", "The JVMF Default messages provide an example of usage of the JVMF"]
+	,["UH-60M", "JVMF"]
+	[true];
```

Comment:
This doesn't look correct.

Reply:
Yep, I see it.  I'm missing a comma.  Does the syntax for cba_fnc_addSetting look correct?

Reply:
I also think the order on where I'm executing the code if off as well, and this function belongs in the "XEH_preInit.sqf"

Reply:
Bool, not an array.